Applications/Games/Subnautica/application.json

```diff
@@ -0,0 +1,2 @@
+  	"name":	"Subnautica",
```

Comment:
This is not a valid json. It must look like this:
```JSON
{
	"name":	"the name",
	"description": "the description"
} 
```

If you load your repository with Phoenicis, you will get an error message reporting this.